src/sentry/static/sentry/app/components/idBadge/slugOverflow.jsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'react-emotion';
+
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+
+const SlugOverflow = styled('span')`
```

Comment:
Default wrapper used when displaying team/project/org slugs. `maxSlugWidth` is currently at `200px`